browser_use/agent/service.py

```diff
@@ -164,7 +165,17 @@ def __init__(
 		# Core components
 		self.task = task
 		self.llm = llm
-		self.controller = controller
+
+		self.allow_yield_to_human = allow_yield_to_human
```

Comment:
this should probably be stored in `AgentSettings`, not just a loose attr on Agent